Contributors-Guide/Guidelines-For-Maintainers.md

```diff
@@ -13,36 +13,40 @@ explicitly called out.
 
 ### Guidelines that Maintainers are expected to adhere to
 
-​1. Ensure qualitative and timely management of patches sent for review.
-
-​2. For merging patches into the repository, it is expected of
-maintainers to:
-
-		a> Merge patches of owned components only.
-		b> Seek approvals from all maintainers before merging a patchset spanning multiple components.
-		c> Ensure that regression tests pass for all patches before merging.
-		d> Ensure that regression tests accompany all patch submissions.
-		e> Ensure that documentation is updated for a noticeable change in user perceivable behavior or design.
-		f> Encourage code unit tests from patch submitters to improve the overall quality of the codebase.
-		g> Not merge patches written by themselves until there is a +2 Code Review vote by other reviewers.
-
-​3. The responsibility of merging a patch into a release branch in
-normal circumstances will be that of the release maintainer's. Only in
-exceptional situations, maintainers & sub-maintainers will merge patches
-into a release branch.
-
-​4. Release maintainers will ensure approval from appropriate
-maintainers before merging a patch into a release branch.
-
-​5. Maintainers have a responsibility to the community, it is expected
-of maintainers to:
-
-		a> Facilitate the community in all aspects.
-		b> Be very active and visible in the community.
-		c> Be objective and consider the larger interests of the community  ahead of individual interests.
-		d> Be receptive to user feedback.
-		e> Address concerns & issues affecting users.
-		f> Lead by example.
+1. Ensure qualitative and timely management of patches sent for review.
+
+1. For merging patches into the repository, it is expected of maintainers to:
```

Comment:
All numbering is set to `1` here. This is surprisingly rendered correctly. Please fix this is upcoming updates to this doc.